obs-browser-plugin.cpp

```diff
@@ -224,6 +224,13 @@ static void BrowserInit(void)
 	path += ".exe";
 	CefMainArgs args;
 #else
+#if defined(__APPLE__) && !defined(BROWSER_DEPLOY)
+	/* Load CEF at runtime as required on macOS */
+	CefScopedLibraryLoader library_loader;
+	if (!library_loader.LoadInMain())
+		return;
```

Comment:
@WizardCM This is actually not so great, because the early return isn't considered by the plugin infrastructure. If you have a suggestion how to handle the case of CEF not being able to load the library and abort/quit gracefully, I can adapt that.

Reply:
Just for my reference, is this a blocker in terms of merging?

Reply:
It's just uncool as you get a blank browser source and the log is spammed with "cannot load" messages, but if the CEF library is in the bundle, it should load it just fine.